Sources/Client/WebSocket/WebSocket.swift

```diff
@@ -10,9 +10,6 @@ import UIKit
 
 /// A web socket client.
 final class WebSocket {
-    /// The time interval to ping connection to keep it alive.
-    static let pingTimeInterval: TimeInterval = 25
-
```

Comment:
It seems you removed this from the v2? :)

Reply:
oh no!